src/GovUk.Education.ExploreEducationStatistics.Content.ViewModels/DataSetFileMetaViewModel.cs

```diff
@@ -3,14 +3,14 @@ namespace GovUk.Education.ExploreEducationStatistics.Content.ViewModels;
 public record DataSetFileMetaViewModel
 {
     public List<string> GeographicLevels { get; init; } = new();
```

Comment:
Can add `required` on all these view model properties and remove any unneeded default value - here and below